src/components/oscal-utils/OSCALLinkUtils.js

```diff
@@ -61,3 +61,22 @@ export function guessExtensionFromHref(url) {
   }
   return detectedExtension;
 }
+
+/**
+ * Finds if a fragment contains a control group.
+ * NOTE: Control groupings will contain two letters, followed by a "-" and a positive integer
+ *
+ * @param frag A given fragment.
+ * @returns A string providing the control group.
+ */
+export function determineControlGroupFromFragment(frag) {
```

Comment:
```suggestion
export function determineControlGroupFromFragment(fragment) {
```

There's no need to shorten this.